lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -612,15 +620,11 @@ def write(chunk)
                                             ts_secs,
                                             ts_nanos)
 
-          # Get fully-qualified trace id for LogEntry "trace" field.
-          fq_trace_id = record.delete(@trace_key)
-          entry.trace = fq_trace_id if fq_trace_id
-
-          span_id = record.delete(@span_id_key)
-          entry.span_id = span_id if span_id
-
-          insert_id = record.delete(@insert_id_key)
-          entry.insert_id = insert_id if insert_id
+          entry.trace = record.delete(@trace_key) if record.key?(@trace_key)
```

Comment:
Should we revert to the previous code?

Reply:
Seems like the same thing. But i reverted anyway.

Reply:
The previous code you've reverted to does one less lookup.